charts/reform-scan-notification-service/values.preview.template.yaml

```diff
@@ -18,3 +23,12 @@ java:
     postgresqlUsername: db_user
     postgresqlPassword: db_password
     postgresqlDatabase: db_database
+servicebus:
+  resourceGroup: reform-scan-aks
+  teamName: "BSP"
+  location: uksouth
+  serviceplan: basic
+  setup:
+    queues:
+      - name: notifications-reform
```

Comment:
we can create correct queue for prs straight away ;)

Reply:
changed